core/contractsdk/cpp/example/trustops/src/trust_counter.cc

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include "xchain/xchain.h"
+#include "xchain/trust_operators/trust_operators.h"
+
+struct Counter : public xchain::Contract {};
+
+DEFINE_METHOD(Counter, initialize) {
+    xchain::Context* ctx = self.context();
+    const std::string& creator = ctx->arg("creator");
+    if (creator.empty()) {
+        ctx->error("missing creator");
+        return;
+    }
+    ctx->put_object("creator", creator);
+    ctx->ok("initialize succeed");
+}
+
+DEFINE_METHOD(Counter, increase) {
```

Comment:
this method just implement store not increase. It is better to rename it to "store" or implement "increase".

Reply:
fixed